### ISSUE TITLE
Setup documentation generation with Orchid

### ISSUE THREAD

**Issue by rock3r** [mpgirro/wien#36]
TODO

**Comment by rock3r** [comment]
@mpgirro Can you expand on this?

**Comment by mpgirro** [comment]
Well I just created this issue as a reminder to play around with [Orchid](https://orchid.run). Around the time I started this lib, I came across some project website that looked nice, and thought I could give Orchid a try here. At least from a quick look it seemed like an easy and automatic way to ship a project website GH pages that includes KDoc in a visual appeal that looks "natural". If this is true, Idk tbh, but this other project's website at least looked like that (I really can't remember anymore what the website was unfortunately).  

Still haven't given it a try ever since, so maybe I am wrong. Nothing is a must. If you have a better idea or good experiences with another tool/setup, I am open.

We can also simply transform this issue into a "setup project website" thing.

**Comment by rock3r** [comment]
No, I don't really have much experience with this kind of things at all, if I'm honest. But thanks for explaining me :)

**Comment by mpgirro** [comment]
I've added a basic setup in the [orchid branch](https://github.com/mpgirro/stalla/tree/orchid), mostly as a proof of concepts and to evaluate the technology for now.

Running on localhost:

```
./gradlew clean orchidServe -PorchidDiagnose=true
```

Current features:

- Generates website based on the markdown files in `src/orchid/resources/`
- Includes (for now) the GitHub Readme content in the homepage
- Includes the Dokka API doc in the website structure.
- Builds a change log page based on individual entries for each version in `src/orchid/resources/changelog/`
- Includes the GitHub repository wiki. Just a dummy page in there for now. Could be used to write the documentation and have the website just mirror the entries.

Current issues: 

- The included Dokka documentation I missing the links between the API pages (classes, packages, etc). I've fixed all errors and warnings reported when running `orchidServe` with `--diagnose`. Yet the issue remains.
